python/cugraph/ktruss/ktruss_subgraph.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import cugraph
+from cugraph.structure.graph import Graph
+
+
+from cugraph.ktruss import ktruss_subgraph_wrapper
+
+
+def ktruss_subgraph(G,k):
+    """
+    Returns the subgraph of the k-truss of a graph for a specific k.
```

Comment:
Please edit https://github.com/rapidsai/cugraph/blob/branch-0.11/docs/source/api.rst so that the doc renders on https://docs.rapids.ai/api/cugraph/nightly/

Reply:
Done.